content/community/guidelines_creating_haiku_distribution.html

```diff
@@ -27,26 +27,38 @@ <h3>Compatibility</h3>
 <li>You must not change or extend the API of the Haiku system libraries in any way.</li>
 <li>You should not change, move, or rename any of the files and folders that are part of the base distribution's "system" folder without good reason.</li>
 <li>Provide a version of AboutSystem that identifies your distribution and contains a disclaimer as described in this document.</li>
-<li>Use the same GCC version as the official distribution (X86 GCC 2.95.3 Hybrid with GCC 4.x alternative GCC libraries) and run the "configure" script with the parameter "--distro-compatibility=compatible".</li>
+<li>Use the same GCC version as the official distribution:
+<ul>
+<li>for 32bit X86: GCC 2.95.3 Hybrid with modern GCC (currently 7.x) alternative GCC libraries</li>
+<li>for 64bit x86: modern GCC (currently 7.x) only</li>
+</ul>
+</li>
+<li>Run the "configure" script with the parameter "--distro-compatibility=compatible".</li>
 </ul>
 
 <h3>Disclaimer</h3>
 
 <p>Please include the following disclaimers in a conspicuous place in your website, your AboutSystem version, and the documentation of your distribution:</p>
 
-<p>&lt;DISTRO NAME&gt; is based on Haiku, but it is not the official distribution from the Haiku Project. For information about the official Haiku project, please visit <a href="http://haiku-os.org">http://haiku-os.org</a>.</p>
+<p>&lt;DISTRO NAME&gt; is based on Haiku, but it is not the official distribution from the Haiku Project. For information about the official Haiku project, please visit <a href="https://haiku-os.org">https://haiku-os.org</a>.</p>
 
 <p>&lt;DISTRIBUTOR NAME&gt; is not associated with the Haiku project. To obtain support for &lt;DISTRO NAME&gt;, please contact &lt;DISTRO SUPPORT INFO&gt;.</p>
 
-<h3>Development and Alpha Releases</h3>
+<h3>Development and Alpha/Beta Releases</h3>
 
-<p>For development and alpha releases please add this to the disclaimer:</p>
+<p>For development and alpha/beta releases please add this to the disclaimer:</p>
```

Comment:
"official releases"

Reply:
You mean "official releases" instead of alpha/beta, or "official alpha/beta releases"?
The disclaimer is not for the official final R1 release.